java/mdsobjects/src/main/java/MDSplus/TreeNode.java

```diff
@@ -599,7 +599,7 @@ public int getNciFlags() throws MdsException
 				flags += flag;
 		return flags;
 	}
-
+/*
```

Comment:
HINT: We use git for versioning. you should not comment old code when there is no lesson to learn.
You need the function signature to wrap around the native entry points. the nid argument is a field of TreeNode instance.

set/getNciFlags should be 
{
resolveNid();
this.setNciFlags(nid, ctxTree.getCtx(), flags); / return this.getNciFlags(nid, ctxTree.getCtx())
}
and keep  this.getNciFlag